CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Unreleased
+
+### Added
+
+- :timezone to navigation protocol
```

Comment:
```suggestion
- `print_to_pdf` now supports a `:timezone` option to set the timezone before navigating with `Emulation.setTimezoneOverride`
```

Reply:
Thank you for this, @andreasknoepfle! Much clearer; I appreciate it.